modules/web-console/frontend/app/components/page-queries/components/queries-notebook/controller.ts

```diff
@@ -2208,13 +2208,8 @@ export class NotebookCtrl {
             const paragraphs = _.get(this, '$scope.notebook.paragraphs');
 
             if (this._hasRunningQueries(paragraphs)) {
-                try {
-                    return Confirm.confirm('You have running queries. Are you sure you want to cancel them?')
-                        .then(() => this._closeOpenedQueries(paragraphs));
-                }
-                catch (err) {
-                    return Promise.reject(new CancellationError());
-                }
+                return Confirm.confirm('You have running queries. Are you sure you want to cancel them?')
+                    .then(() => this._closeOpenedQueries(paragraphs));
```

Comment:
I don't see how this is related to the task. Commit messages tell nothing useful about this change either.

Reply:
This code don't throw exceptions, just return promise.